pipelines/rh-advisories/README.md

```diff
@@ -22,6 +22,9 @@ the rh-push-to-registry-redhat-io pipeline.
 | taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
+## Changes in 0.13.0
+- Add tasks `collect-cosign-params` and `rh-sign-image-cosign` to sign images by cosign
```

Comment:
Maybe add a note saying that `rh-sign-image-cosign` is only run if sign.cosignSecretName is set in the data file?

(Same for the other pipeline.)

Reply:
Added.

Reply:
Thanks.

Reply:
The when condition is awesome, nicely done